### PR TITLE
Feat/oak pupil journey programme options

### PR DESCRIPTION
**Pull request by benprotheroe**
# How to review this PR

_Leave this text block for the reviewer_

- Check [component hierarchy](https://miro.com/app/board/uXjVNnKBgyk=/?share_link_id=59445593794) is followed correctly
- Check the design [Heuristics](https://components.thenational.academy/?path=/docs/docs-howtodesigncomponents--docs#heuristics-for-component-design) have been followed
- Check [naming conventions](https://components.thenational.academy/?path=/docs/docs-namingconventions--docs) have been applied
- Check for these gotchyas:
  - Missing exports for Oak components
  - Accidental export of Internal components
  - Snapshots of unexpected components have been modified
  - Circular dependencies
  - Code duplication (via not using base components)
  - Non-functional storybook for this or related components
  - Sensitve files changed eg. atoms, or style tokens
  - Relative imports
  - Default exports

# Add your PR description below

create a component which handles the inner section on the programmes options page

### CODE REVIEW

File: src/components/organisms/pupil/OakPupilJourneyLayout/OakPupilJourneyLayout.tsx [hunk 1]
```diff
@@ -62,6 +62,10 @@ export const OakPupilJourneyLayout = ({
         return "bg-decorative4-very-subdued";
       case sectionName === "unit-listing" && phase === "secondary":
         return "bg-decorative3-very-subdued";
+      case sectionName === "tier-listing" && phase === "primary":
```
Comment (benprotheroe) [inline src/components/organisms/pupil/OakPupilJourneyLayout/OakPupilJourneyLayout.tsx:65]: i think you left this out of the last pr

File: src/components/organisms/pupil/OakPupilJourneyProgrammeOptions/OakPupilJourneyProgrammeOptions.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        $pa={"inner-padding-xl"}
+        $borderColor={backgroundColor}
+      >
+        <OakFlex
+          $pv={"inner-padding-xl"}
+          $flexDirection={"column"}
+          $alignItems={"center"}
+          $gap={"space-between-l"}
+        >
+          {optionTitleSlot}
```
Comment (benprotheroe) [inline src/components/organisms/pupil/OakPupilJourneyProgrammeOptions/OakPupilJourneyProgrammeOptions.tsx:63]: this is for the bit inside the options list box thing, its the title of the option ie either tier or examboard and then the tooltip

**Comment by RBrownsett** [comment]
@benprotheroe which ticket does this relate to? Also I can seem to see any testing links.

**Comment by messmike** [comment]
@benprotheroe just checking if the info button for the exam board programme option and the tier programme option will be added here or to the page(s)? E.g:
Screenshot 2024-05-13 at 4 43 37 pm
Screenshot 2024-05-13 at 4 43 49 pm
Screenshot 2024-05-13 at 4 43 54 pm
Screenshot 2024-05-13 at 4 43 31 pm

**Comment by benprotheroe** [comment]
> @benprotheroe just checking if the info button for the exam board programme option and the tier programme option will be added here or to the page(s)? E.g: Screenshot 2024-05-13 at 4 43 37 pm Screenshot 2024-05-13 at 4 43 49 pm Screenshot 2024-05-13 at 4 43 54 pm Screenshot 2024-05-13 at 4 43 31 pm

yeah the tooltip will be added in OWA

**Comment by benprotheroe** [comment]
> @benprotheroe which ticket does this relate to? Also I can seem to see any testing links.

doesnt have a ticket per se but its needed in order to finish PUPIL-680

**Comment by RBrownsett** [comment]
@benprotheroe there has been accessibility violation - though from your code I couldn't see that you used aria-bust setting? 

image

**Review by messmike — APPROVED** [review]

**Comment by RBrownsett** [comment]
@benprotheroe I think the updating the heading from h2 to h3 has caused the violations from one to us now having two due to it not being semantic? 
image

**Comment by RBrownsett** [comment]
@benprotheroe just checking that that the text **Primary Lessons** and the subject icon can be updated depending on the subject that is being displayed? 

image

**Comment by benprotheroe** [comment]
> https://deploy-preview-172--lively-meringue-8ebd43.netlify.app

yeah the subject data is pipedmin via owa

**Review by RBrownsett — APPROVED** [review]
Looks good and the accessibility issue has now been fixed, great work :-)

**Review by kimon-satan — APPROVED** [review]

**Pull request merged**